packages/flutter/test/material/dialog_test.dart

```diff
@@ -130,7 +130,7 @@ void main() {
     await tester.pumpAndSettle();
 
     final Material materialWidget = _getMaterialFromDialog(tester);
-    expect(materialWidget.color, theme.colorScheme.surface);
+    expect(materialWidget.color, theme.colorScheme.surfaceContainerHigh);
```

Comment:
Can we also correct the test message to "M3 Dialog background defaults to ColorScheme.surfaceContainerHigh"? Probably can also update the `colorScheme` part:)

Reply:
I updated the title and color scheme but "M3" isn't necessary since we no longer label M3 tests, it's default.